test/glue/candlepin/provider_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'katello_test_helper'
+require 'support/candlepin/owner_support'
+
+module Katello
+  class GlueCandlepinProviderTestBase < ActiveSupport::TestCase
+    include VCR::TestCase
+
+    def setup
+      User.current = User.find(FIXTURES['users']['admin']['id'])
+    end
+  end
+
+  class GlueCandlepinProviderTest < GlueCandlepinProviderTestBase
+    def setup
+      super
+      @org = CandlepinOwnerSupport.create_organization('GlueCandlepinProviderTest', 'GlueCandlepinProviderTest')
+      @raw_attrs = Katello::ProductTestData::PRODUCT_WITH_CP_CONTENT.dup
+    end
+
+    def teardown
+      super
+      CandlepinOwnerSupport.destroy_organization(@org)
+    end
+
+    def test_import_products
```

Comment:
Updating these tests to actually go through the provider and not right through the Glue model.